content/technology/terms/_index.md

```diff
@@ -56,7 +56,7 @@ Currently these microservices are part of Platform: Authorization, Authenticatio
 
 Altinn Platform is running in its own Kubernetes cluster.
 
-See Altinn Platform [solution architecture](https://docs.altinn.studio/architecture/solution/altinn-platform/), [application architecture](https://docs.altinn.studio/architecture/application/altinn-platform/)
+See Altinn Platform [solution architecture](/architecture/solution/altinn-platform/), [application architecture](/architecture/application/altinn-platform/)
```

Comment:
Use `../solutions/altinn-platform/` and `../architecture/components/application/solution/altinn-platform/` instead.

Sorry for the mess, we're planning to clean up and simplify the architecture-part of the doc, but have not found the time yet. Very nice of you to try to fix these pesky absolute urls 🙏

Reply:
I had some issues with the search taking me to the full url, so I searched for every instance and tried to fix them. It didn't help for the search so I guess that's related to the 'baseUrl' config that I can't see how to fix for docker only.